app/models/agenda-item-treatment.js

```diff
@@ -5,8 +5,8 @@ export default class AgendaItemTreatment extends Model {
   @attr('datetime') created;
 
   @belongsTo('decision-activity') decisionActivity;
+  @belongsTo('news-item') newsItem;
```

Comment:
Before this relationship had `serialize: false`. I'm not sure why. I changed it here because the inverse is now a `hasMany`. Ember doesn't serialize many-to-one relationships by default (you need to save on the one-to-many side to persist the relationship), so I made it serializable on this side.

We could also make the many-to-one side serializable explicitly by adding `serialize: true`, but I'm not sure why it was disabled on this side to begin with, and if something could break from serializing the many-to-one side.

Reply:
It was added as a concurrency fix in https://kanselarij.atlassian.net/browse/KAS-3453.
What happened was that user 1 was creating a newsletter while user 2 uploaded a report (then attached to treatment, now somewhere else) so user 2  saved `newsletter` as `null` on treatment, disconnecting the `newsletter` created by user 1.
You can try the steps to reproduce to check if it still occurs. instead of uploading a report, try also creating a conflicting newsletter or just saving the model

I haven't really looked at the other code just yet but I'm lost as to why a newsitem would have many treatments. We create 1 treatment, use that as a reference to connect to many agendaitem, 1 newsitem and 1 decisionActivity.
If the `has-many` of `treatments` is defined in the themis model I would argue that this is incorrect.
Since we currently implement one-to-one, when will the has many ever come in to play?

Reply:
Sorry for the confusion. Relation between treatment and newsitem is indeed 1-on-1 as @ValenberghsSven says. The story description hasn't been updated after the data migration where we split treatment and decision-activity. Before that migration the relation used to be has-many.

Reply:
Removal of unnecessary `serialize: false` and `inverse: null` options will be tackled in future stories, so maybe best to keep them here.